ntc_templates/templates/alcatel_sros_show_service_sap-using.textfsm

```diff
@@ -22,6 +22,7 @@ SAPS
   ^=+
   ^Number\s+of
   ^\s*$$
+  ^\S+
```

Comment:
What is the reason for this change? Can you update the test case or add a test case with data the exemplifies the issue this is fixing?